driver-scala/build.gradle

```diff
@@ -17,6 +17,13 @@
 description = "A Scala wrapper of the MongoDB Reactive Streams Java driver"
 archivesBaseName = 'mongo-scala-driver'
 
+configurations {
+    integrationTestImplementation {
+        extendsFrom testImplementation
+        canBeResolved = true
```

Comment:
This should fix intellij - also it seems extending from testRuntime wasn't needed.

Reply:
Indeed, it did.